draft-ietf-quic-transport.md

```diff
@@ -367,6 +367,10 @@ version negotiation and 1-RTT keys are established.
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                          Payload (*)                        ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                        [FNV Hash(64)]                         +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
Maybe my comment has caused confusion here. My apologies.

The PR proposes to add a field named "FNV Hash" _after_ Payload. So the location at where the hash should be stored might not be at stake.

OTOH, simply adding a "FNV Hash" field contradicts with the expectation that the hash value will be part of the Payload. For example, 5.4 states:
> The payload of cleartext packets also includes an integrity check, which is described in [QUIC-TLS].

The case is also the same of the AEAD tag.

So while I am not against creating a field to mark where the hash and tag needs to be stored (though I am not sure if it is necessary to do so), I think that such a PR should also require adjustments to the text of the drafts as well.